### ISSUE TITLE
where can I download Cosaliency image pair dataset?

### ISSUE THREAD

**Issue by adityaksai** [HzFu/SACS_TIP2014#1]
The original website is not working.
http://ivipc.uestc.edu.cn/hlli/projects/cosaliency.html
 please help me.

**Comment by HzFu** [comment]
Yes, it seems the web invalid. But I am not the author of the dataset.
Please email the dataset authors to obtain the new link.

**Comment by adityaksai** [comment]
Thank you so much for your reply. yes,I did. Do you have any copy of the dataset?
